lib/actions/deploy.coffee

```diff
@@ -186,21 +193,20 @@ module.exports =
 							{ image: imageName, log: '' }
 					.then ({ image: imageName, log: buildLogs }) ->
 						logs = buildLogs
-						Promise.join(
-							dockerUtils.bufferImage(docker, imageName, tmpPath)
+						Promise.all [
```

Comment:
I personally do find this slightly clearer to read here, I wasn't very keen on the `Promise.join` when I wrote it, this at least makes it obvious that the `performUpload` is the function.